src/main/java/seedu/opus/ui/CommandBox.java

```diff
@@ -194,8 +194,8 @@ private void browseToPrecedingCommand() {
      */
     private static class AutocompleteTrie extends Trie {
 
-        private static final String[] COMMANDS = { "add", "delete", "edit", "mark", "unmark", "schedule", "list",
-            "help", "find", "undo", "redo", "clear" };
+        private static final String[] COMMANDS = { "add", "delete", "edit", "mark", "schedule", "list",
```

Comment:
Wouldn't it be better to use AddCommand.COMMAND_WORD etc instead of magic strings?